### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by phelian23**
In this PR
- Styling is added

### CODE REVIEW

**Comment by omar-labana** [comment]
Hi @phelian23, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, where the linters are not passing. Please fix the linter errors.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/components/addNew.js [hunk 1]
```diff
@@ -22,26 +24,37 @@ const AddNew = () => {
...
+        setInputValues({
+          errMsg: 'Please select category',
+        });
+      }
+    } else {
+      setInputValues({
+        errMsg: 'Please Add Title',
+      });
+    }
   };
```
Comment (Emmyn5600) [inline src/components/addNew.js]: - [x] When I clicked on the `add new book` without filling out the title and category, I got this error. please fix it.

setup-11

File: src/styles/addNew.module.css [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  padding: 0.801rem 1.188rem 0.886rem 1.375rem;
+  border-radius: 3px;
+  background-color: #0290ff;
+  border: 0;
+  font-family: roboto slab, sans-serif;
+  font-size: 0.813rem;
+  font-weight: bold;
+  letter-spacing: 0.5px;
+  color: #fff;
+}
```
Comment (Emmyn5600) [inline src/styles/addNew.module.css:59]: - [x] Let try to match with the design provided.

---------------------I Original

setup-9

----------------------| Yours

setup-12

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
Hi @phelian23 

Great work on making the project requirements 👏 
You've done well implementing some of the project requirements, but there are still some which aren't addressed yet, please work on it and ask for another review.


 ## Required Changes ♻️
 
- _Check the comments under the review._

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by Lameck1 — APPROVED** [review]
Hi, @phelian23,

You've done well working on the concerns of the previous review 👍🏼.
The tasks for this milestone are therefore successfully implemented 💯.

Your project is now complete, you are free to merge this branch :shipit:

Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**